src/Classes/RichTextViewController.swift

```diff
@@ -435,17 +484,233 @@ public class RichTextViewController: UIViewController {
             textView.selectedRange = range
         }
     }
+    
+    // MARK: Font Adjustments
+    
+    private func applyFontAttribute(font: UIFont) {
+        guard let text = textView.attributedText else { return }
```

Comment:
This should be named better, since its an attributedString, maybe call it `attributedString`